src/dataIntegration/transcriptomics/formatGenes.m

```diff
@@ -0,0 +1,57 @@
+function modelNew=formatGenes(model)% TO DO - need to be adapated depending on the gene format used for cobratoolboxv3
+%Gets rid of the Entrez suffixes ".x" or preffixes "HGNC:" in the gene identifiers 
+%used in the model and updates the GPR rules with the new gene format.
+%
+%INPUTS
+%
+%   model       model structure containing genes in the form "entrez#.x"
+%               (e.g. case of recon1) or in the form "HGNC:entrez#" (e.g. case of recon2) 
+%   
+%OUTPUTS 
+%
+%   modelNew    model structure with formatted genes (all genes are just
+%               "enrez#" (string))
+%
+% S. Opdam & A. Richelle May 2017
+    
```

Comment:
This is something I would leave to the user. If the model uses HGNC-IDs and his data is for Entrez-IDs, tehe user should do the conversion, since conversion is not always direct, and we can't know future nomenclatures (so it wouldn't be future proof anyways).